doc/bin/print_rotonda_std_doc.rs

```diff
@@ -0,0 +1,6 @@
+use rotonda::common::roto_runtime::rotonda_roto_runtime;
```

Comment:
Could that module path be simplified to `rotonda::common::roto::runtime::init` maybe? It duplicates each of roto, rotonda and runtime which feels very unnecessary.

Reply:
Good sugestion, I think this is still a bit in flux (there's a roto.rs, rote_new.rs and a roto_runtime.rs in there currently. Will relay this question to the author